src/m365/commands/login.ts

```diff
@@ -151,6 +197,88 @@ class LoginCommand extends Command {
     await this.initAction(args, logger);
     await this.commandAction(logger, args);
   }
+
+  private tokensForMultipleTenants(accessTokenValue: string | string[] | undefined, tenantValue: string | undefined): boolean {
+    const accessTokens = typeof accessTokenValue === "string" ? [accessTokenValue] : accessTokenValue as string[];;
```

Comment:
Won't this fail if accessToken is undefined?

Reply:
I added that as a check to the refinement condition, I should remove `| undefined` here...